src/components/DropdownMenu/README.md

```diff
@@ -395,24 +395,31 @@ LANDING_BLOCK-->
 
 <!--/GITHUB_BLOCK-->
 
+## Switcher properties
```

Comment:
This section breaks in the current section, it breaks the following paragraph away from it

Let's move this section to the very bottom of the file, under a third-level `SwitcherProps` heading after the `DropdownMenuItem` section. Here, we could add a reference to the `SwitcherProps` section:

```md
It can be any function that returns a React component (or any `(props: SwitcherProps) => React.ReactNode` in the TypeScript terms, see [`SwitcherProps`](#switcherprops) below).
```